src/components/shared/tests/ResultList.spec.js

```diff
@@ -6,7 +6,7 @@ import React from "react";
 import { shallow } from "enzyme";
 import ResultList from "../ResultList";
 
-const selectItem = jest.genMockFunction();
+const selectItem = jest.fn();
```

Comment:
`genMockFunction` was removed and was an undocumented API. Removing it from our examples and code.

https://github.com/facebook/jest/pull/6173